### DISCUSSION TITLE
Lasso and box selection

### DISCUSSION THREAD

**Discussion by bkgmc** [LayTec-AG/Plotly.Blazor#429] [Q&A]
Hi everyone,
thank you for providing this library! 
There's one thing I was not able to figure out by myself:
Could anybody please give me a hint on how to use the lasso and box selection tools in a scatter plot?
I've been playing around with the ClickMode, but I do not understand how to access the list of selected points. There does not seem to be a related event. What am I missing?
Thanks for your support and have a nice weekend!

**Comment by sean-mcl** [comment]
Hi,

[here](https://laytec-ag.github.io/Plotly.Blazor/selected) is the example. :)

**Reply by GiorgioMauro** [reply]
When i open your link i receive this message "Sorry, there's nothing at this address."

**Reply by sean-mcl** [reply]
> When i open your link i receive this message "Sorry, there's nothing at this address."

Mhm, can you open this link? https://laytec-ag.github.io/ 

On left is the "Selected" example, you may have to zoom out to see it properly.

**Reply by sean-mcl** [reply]
> When i open your link i receive this message "Sorry, there's nothing at this address."

Otherwise, here is the example as code.

https://github.com/LayTec-AG/Plotly.Blazor/blob/main/Plotly.Blazor.Examples/Components/Selected.razor

**Reply by GiorgioMauro** [reply]
Hey @sean-mcl,

 That’s exactly what I wanted. Thank you for your support! 😊

Have a good day!
